src/PostgREST/Parsers.hs

```diff
@@ -171,7 +171,7 @@ pOrderTerm =
   <|> OrderTerm <$> pField <*> pure Nothing <*> pure Nothing
 
 pLogicTree :: Parser LogicTree
-pLogicTree = try (pNotLogicOp <*> (char '{' *> pLogicTree) <*> (char ',' *> pLogicTree <* char '}'))
+pLogicTree = try (pNotLogicOp <*> (char '(' *> pLogicTree) <*> (pDelimiter *> pLogicTree <* char ')'))
```

Comment:
why the switch to `.` wasn't the discussion about `+`? although i don't see whats wrong with `,`
`id.eq.1.id.eq.2` is a very ugly format.
PS: the parser also needs to support spaces before/after the separator or maybe the space should be the separator like

`/entities?or=(id.eq.1 id.eq.2)&select=id,name` but if you do it like this, there is in inconsistency in regard to `select`

so again, i don't see why it's not implemented like this
`/entities?or=(id.eq.1, id.eq.2,id.eq.3)&select=id,name`